lib/internal/util.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const prefix = '(node) ';
+const prefix = `(${process.release.name}:${process.pid}) `;
```

Comment:
I'd like someone with more knowledge to weigh in about possibly using `process.title` instead. Maybe cc @bnoordhuis?